### PR TITLE
Replace implementation of the pull to refresh pattern

### LINKED ISSUE #186: Replace old pull to refresh with new one

### PR DESCRIPTION
**Pull request by HenriqueRocha**
While working in the implementation of the new navigation drawer, I noticed by accident that the app is using the pull to refresh pattern, which looks really ugly with the new action bar.

Currently, the app is using Chris Bane's Pull To Refresh library, which is no longer maintained.

To close my opened issue #186, I've implemented the necessary changes to replace the library with Android's SwipeRefreshLayout.

Fixes #186

### CODE REVIEW

File: app/src/main/java/org/gdg/frisbee/android/fragment/GdlListFragment.java [hunk 1]
```diff
@@ -98,28 +85,25 @@ public void onActivityCreated(Bundle savedInstanceState) {
...
-        mAdapter = new GdlAdapter(getActivity(), ((GdgActivity)getActivity()).getGoogleApiClient());
+        mAdapter = new GdlAdapter(getActivity(), ((GdgActivity) getActivity()).getGoogleApiClient());
         setListAdapter(mAdapter);
 
         setIsLoading(true);
 
         fetchContent(false);
 
-        if(getArguments().getBoolean("active")) {
+        if (getArguments().getBoolean("active")) {
```
Comment (ViliusKraujutis) [inline app/src/main/java/org/gdg/frisbee/android/fragment/GdlListFragment.java]: Such constants as `active` needs to be extracted.

Comment (HenriqueRocha) [inline app/src/main/java/org/gdg/frisbee/android/fragment/GdlListFragment.java]: Should that be part of this PR? I didn't change that code. The only reason why it appears in the diff is because of the reformatting. I'm going to do some clean up in that class anyway so I believe it should go into a different PR.

Comment (ViliusKraujutis) [inline app/src/main/java/org/gdg/frisbee/android/fragment/GdlListFragment.java]: Yes, it should go to a separate PR. Just an observation. I'll raise new issue for that.

Comment (ViliusKraujutis) [inline app/src/main/java/org/gdg/frisbee/android/fragment/GdlListFragment.java]: Added new issue for that: https://github.com/gdg-x/frisbee/issues/189

**Comment by mauimauer** [comment]
Don't put too much work in the Gdl* classes. Those are legacy. GDL (Google Developer Live) has been dead for ages. So maybe remove the classes altogether in a new PR

**Comment by HenriqueRocha** [comment]
@mauimauer Okay, I'll do that as soon as possible. :)

**Comment by stefanhoth** [comment]
@HenriqueRocha Could you add a screenshot or a gif of the visual change?

**Comment by HenriqueRocha** [comment]
Is this good enough?

pull_to_refresh

The reason why "Berlin Android" is in black is because I didn't have the chapter list when I implemented the new action bar. That should be dealt with in a new PR anyway.

**Comment by stefanhoth** [comment]
Wonderful!

**Comment by stefanhoth** [comment]
This looks good but I want to try it out first before I would merge it.

**Comment by HenriqueRocha** [comment]
Yes, go ahead. I've worked on a fix for a bug that I introduced that crashes the app when you try to go to the second screen of the start flow. I'm working on setting up Espresso to test that it actually works correctly and I'll push the code as soon as it's finished so you might want to wait a bit before trying this out so that you don't have to fix the bug yourself.

**Comment by friedger** [comment]
@HenriqueRocha I close it until fixed. Feel free to reopen it.

**Comment by HenriqueRocha** [comment]
The fix is in PR #194 which I pulled today.

**Comment by stefanhoth** [comment]
@HenriqueRocha Sorry, couldn't get to merge it earlier. Could you rebase it so it becomes mergable again?

**Comment by HenriqueRocha** [comment]
@stefanhoth Done. The conflict was in the GdlListFragment because it was removed in the PR I did that removed the GDL related stuff. Once this is merged we'll be another step forward to materializing the app. Work on replacing the ViewPagerIndicator with the new fancier SlidingTabLayout is finished, I'll create a PR tomorrow and code for the new navigation drawer has already been started. :+1:
